update_docker_image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+# DEVELOPER: update this field anytime you make a new docker image
+VERSION="2.1"
+BUILD_TAG="broadinstitute/horsefish:v${VERSION}"
+
+echo "using build tag: ${BUILD_TAG}"
+
+if docker manifest inspect $BUILD_TAG > /dev/null ;
+then
+    echo "Docker image already exists. Do you want to overwrite this tag?."
+    read -p "Press [Enter] to overwrite or [Ctrl+C] to exit and change the version number."
+fi
+
+echo "Docker image does not exist. Building new image."
```

Comment:
don't you want this in an else statement? looks like you could get here even after confirming you want to overwrite the existing tag, right?

Reply:
I thought so too (I've never used bash for conditional logic) but it seems the else is implicit. (https://www.thegeekstuff.com/2010/06/bash-if-statement-examples/)
certainly is less typing anyway
BUT my echo statement is from when I had started to write the else, so that's confusing. I'll change the wording there.

Reply:
hmm, I'm not sure I follow about the else being implicit - that would suggest that you'll only get to this echo statement if the image doesn't already exist (the condition of the if statement). but you'll get here even if the image exists but the user has pressed Enter, right?